### PR TITLE
IDEN-36 Sorts out tags and category

### PR DESCRIPTION
**Pull request by chowdud**

### CODE REVIEW

File: checks/http/atlassian.rb [hunk 1]
```diff
@@ -205,7 +205,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[SaaS Development SaaS],
+              tags: %w[SaaS Development],
```
Comment (duartemateus) [inline checks/http/atlassian.rb]: array styling

File: checks/http/cpanel.rb [hunk 2]
```diff
@@ -46,7 +46,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[COTS Administrative],
+              tags: %w[COTS Hosting Administrative],
```
Comment (duartemateus) [inline checks/http/cpanel.rb]: array styling

File: checks/http/dynatrace.rb [hunk 3]
```diff
@@ -7,10 +7,11 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[APM JavaScript],
+              tags: %w[Analytics APM JavaScript],
```
Comment (duartemateus) [inline checks/http/dynatrace.rb]: array styling

File: checks/http/flywheel.rb [hunk 4]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[CMS SaaS],
+              tags: %w[CMS SaaS PaaS],
```
Comment (duartemateus) [inline checks/http/flywheel.rb]: array styling

File: checks/http/intercom.rb [hunk 5]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[JavaScript Support],
+              tags: %w[JavaScript Support CRM SaaS],
```
Comment (duartemateus) [inline checks/http/intercom.rb]: array styling

File: checks/http/new_relic.rb [hunk 6]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[APM JavaScript],
+              tags: %w[Analytics Cloud SaaS JavaScript],
```
Comment (duartemateus) [inline checks/http/new_relic.rb]: array styling

File: checks/http/sentry.rb [hunk 7]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'service',
-              tags: %w[JavaScript SaaS APM],
+              tags: %w[JavaScript APM],
```
Comment (duartemateus) [inline checks/http/sentry.rb]: array styling

File: checks/http/synacor.rb [hunk 8]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'application',
-              tags: %w[Productivity COTS],
+              tags: %w[Productivity Cloud SaaS],
```
Comment (duartemateus) [inline checks/http/synacor.rb]: array styling

File: checks/http/sophos.rb [hunk 9]
```diff
@@ -30,7 +30,7 @@ def generate_checks(url)
               vendor: 'Sophos',
               product: 'UTM',
               website: 'https://www.sophos.com/en-us/products/unified-threat-management.aspx',
-              description: 'devuce configuration - unique string',
+              description: 'device configuration - unique string',
```
Comment (duartemateus) [inline checks/http/sophos.rb]: should it be "Device" with capital D or even even changing the description to be inline with the new checks?

File: checks/http/tplink.rb [hunk 10]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'application',
-              tags: %w[Networking COTS Router],
+              tags: %w[Networking Router],
```
Comment (duartemateus) [inline checks/http/tplink.rb]: array styling

File: checks/http/communigate.rb [hunk 11]
```diff
@@ -1,37 +1,35 @@
...
-      }
-    ]
+  module Ident
+    module Check
+      class Communigate < Intrigue::Ident::Check::Base
+        def generate_checks(url)
+          [
+            {
+              type: 'fingerprint',
+              vendor: 'Communigate',
```
Comment (duartemateus) [inline checks/http/communigate.rb]: Should this be "ComuniGate" like the product?

File: checks/http/communigate.rb [hunk 12]
```diff
@@ -1,37 +1,35 @@
...
+              category: 'application',
+              tags: ['COTS', 'Marketing', 'Web Server'],
+              product: 'CommuniGate',
+              website: 'http://www.stalker.com/CommuniGatepro/',
+              description: 'server header',
```
Comment (duartemateus) [inline checks/http/communigate.rb]: Should description be changed to be inline with the rest of the new checks?

File: checks/http/centos.rb [hunk 13]
```diff
@@ -27,7 +27,7 @@ def generate_checks(url)
             {
               type: 'fingerprint',
               category: 'application',
-              tags: ['Administrative', 'Login Panel'],
+              tags: ['Administrative', 'Management', 'Hosting', 'Login Panel'],
```
Comment (shpendk) [inline checks/http/centos.rb]: Why did we chose `Hosting` for this?

File: checks/http/moodle.rb [hunk 14]
```diff
@@ -6,7 +6,7 @@ def generate_checks(url)
           [
             {
               type: 'fingerprint',
-              category: 'application',
+              category: 'service',
```
Comment (shpendk) [inline checks/http/moodle.rb]: Why did we decide this is a service? Afaik moodle is an application/cms

File: checks/http/plesk.rb [hunk 15]
```diff
@@ -7,7 +7,7 @@ def generate_checks(url)
             {
               type: "fingerprint",
               category: "service",
-              tags: ["COTS"],
+              tags: ["SaaS"],
```
Comment (shpendk) [inline checks/http/plesk.rb]: is plesk just SaaS nowadays? I feel like it always had an on-premise installation option

**Review by shpendk — APPROVED** [review]
Few changes (see inline), rest looks good 👍

**Review by shpendk — APPROVED** [review]
nice work

**Pull request merged**